src/pybind/mgr/orchestrator/module.py

```diff
@@ -419,7 +420,9 @@ def ukn(s):
                     1: 'running',
                     None: '<unknown>'
                 }[s.status]
-                if s.status == 1 and s.started:
+                if s.status_desc:
+                    status += f' ({s.status_desc})'
+                elif s.status == 1 and s.started:
                     status += ' (%s)' % to_pretty_timedelta(now - s.started)
```

Comment:
I think this means we won't see the time for most instances, since status_desc is normally set to 'running', 'stopped', etc.

Reply:
Right. `stopped (stopped)` doesn't make much sense. I'll have a look

Reply:
better?

Reply:
yeah that should work!